### ISSUE TITLE
Learning rate matters

### ISSUE THREAD

**Issue by iskandr** [openvax/mhcflurry#64]
We're currently using adaptive SGD (mostly RMSprop) and hoping that the default parameters work for us. Many recent (and not so recent) deep learning papers have a schedule for decreasing the learning rate over time, often phrased something like "halve the learning rate every 50 epochs". Here's an extreme case from [Densely Connected Convolutional Networks](https://arxiv.org/abs/1608.06993) (learning rate drop by 10x accompanied with major decrease in loss): 

![DenseNet loss vs. epochs](https://www.dropbox.com/s/vx642vno1a7fbe4/Screenshot%202016-10-13%2019.44.38.png?dl=1)

I imagine the following three parameters would be a useful addition to the cross-validation loop used to determine optimal hyperparameters:
- initial learning rate
- epochs until LR decay
- LR decay factor

**Comment by timodonnell** [comment]
I played around with learning rate informally and did not see opportunities for easy wins, although I didn't explore the full space of decay schedules etc. Closing for now.
